Automattic/wp-calypso
# Hosting Config: Use `<ClipboardButtonInput />` instead of custom

Fixes https://github.com/Automattic/wp-calypso/issues/72966

## Proposed Changes

Switches to `<ClipboardButtonInput />` instead of using custom code:

<img width="762" alt="image" src="https://user-images.githubusercontent.com/36432/217958717-8aad8fbe-6475-460f-83f7-a292f67693eb.png">

## Testing Instructions

1. Navigate to Hosting Config.
2. Verify each of the input fields appear as expected. Compare your local environment to production.
3. Verify the 'Copy' feature works for each input field.
